### DISCUSSION TITLE
Questions about executable system calls

### DISCUSSION THREAD

**Discussion by skandbug** [aquasecurity/tracee#2946] [Questions and Help]
Thank you for your quick answer. #2943

However, if you trace an executable program, the system calls are not fixed.
How do we get the sequence of system calls for an application?
e.g,:
![image](https://user-images.githubusercontent.com/52226009/229138248-3734cc78-15b5-4dec-b4b3-ad616a9d7241.png)

Is it possible to get these system calls and the time of execution of each system call using tracee?

Thank you very much!

**Comment by skandbug** [comment]
in #2943

Depending on the scenario you gave, it may be possible to get the specified system calls, but, in most application executions, she may generate a lot of system calls (you gave 360 system calls). Being able to show all of them in the result you gave would make it easier for us to use.

**Comment by itaysk** [comment]
please use the "Discussions" Tab for general support questions. I'm converting this and the other issue you created to discussions.

**Reply by skandbug** [reply]
OK！
Thank you for reminder.

**Comment by yanivagman** [comment]
See my answer here: https://github.com/aquasecurity/tracee/discussions/2945#discussioncomment-5490048
